picasso/src/main/java/com/squareup/picasso3/MatrixTransformation.kt

```diff
@@ -111,8 +112,9 @@ internal class MatrixTransformation(private val data: Request) : Transformation
         }
 
         // EXIf interpretation should be done before cropping in case the dimensions need to
-        // be recalculated
-        if (exifOrientation != 0) {
+        // be recalculated; SDK 28+ uses the ImageDecoder API which automatically applies
+        // the EXIf orientation to the bitmap
+        if (exifOrientation != 0 && VERSION.SDK_INT < 28) {
```

Comment:
nit: would this concept be better wrapped in a:

```kotlin
fun shouldPerformExifOrientation() = VERSION.SDK_INT < 28

..

if (exifOrientation != 0 && shouldPerformExifOrientation()) {
```

(as opposed to an inline comment)